docs/quickstart/includes/publish-manage.md

```diff
@@ -1,17 +1,18 @@
-From your profile on nuget.org, select **Manage Packages** to see the one you just published. You also receive a confirmation email. Note that it might take a while for your package to be indexed and appear in search results where others can find it. During that time your package page shows the message below:
+When your package successfully publishes, you receive a confirmation email. To see the package you just published, on [nuget.org](https://www.nuget.org/), select your user name at upper right, and then select **Manage Packages**.
 
-![This package has not been indexed yet. It will appear in search results and will be available for install/restore after indexing is complete.](../media/QS_Create-03-NotIndexed.png)
+> [!NOTE]
+> It might take awhile for your package to be indexed and appear in search results where others can find it. During that time, your package appears under **Unlisted Packages**, and the package page shows the following message:
+> 
+> ![This package has not been indexed yet. It will appear in search results and will be available for install/restore after indexing is complete.](../media/QS_Create-03-NotIndexed.png)
 
-And that's it! You've just published your first NuGet package to nuget.org that other developers can use in their own projects.
+You've now published a NuGet package to nuget.org that other developers can use in their projects, but the sample package uses an empty class library that isn't useful. To *unlist* this test package and hide it from search results:
```

Comment:
the nuget.org FAQ has an entry about test packages: https://docs.microsoft.com/en-us/nuget/nuget-org/nuget-org-faq#is-it-recommended-to-upload-my-test-packages-to-nuget-org

it suggests using int.nugettest.org, rather than nuget.org.  Perhaps the quickstart should follow the same recommendation?

Reply:
Added a note.